scripts/stork.sh

```diff
@@ -124,7 +124,7 @@ if [[ "$kind" == "app" ]]; then
     cp $MOOSE_DIR/.gitignore $dir/
 
     dir="$PWD/$dir"
-    (cd $dir && git init && git add * .clang-format .gitignore && git commit -m"Initial files")
+    (cd $dir && git init -b main && git add * .clang-format .gitignore && git commit -m"Initial files")
```

Comment:
```suggestion
    (cd $dir && git init && git branch -m main && git add * .clang-format .gitignore && git commit -m"Initial files")
```
This change is more backwards compatible with `git < 2.28`, which did not have a `-b` flag for `git-init`.